src/msgpack.lua

```diff
@@ -0,0 +1,220 @@
+local bit = require 'bit'
```

Comment:
It breaks our lua styleguide (https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#idioms-and-patterns)

```
Avoid these kind of constructions:

<func>’<name>’ (strongly avoid require’..’)
```

Reply:
Yup, finally have read these guidelines - will fix